packages/ringcentral-integration/modules/Storage/index.js

```diff
@@ -3,29 +3,41 @@ import StorageBase from '../../lib/StorageBase';
 import loginStatus from '../Auth/loginStatus';
 import moduleStatuses from '../../enums/moduleStatuses';
 
+const DEFAULT_ALLOW_INACTIVE_TABS_WRITE = false;
```

Comment:
It will be better to allow write by default. So we doesn't change default behavior.